script.js

```diff
@@ -0,0 +1,55 @@
+const $ = document;
+
+const bookArray = JSON.parse(localStorage.getItem('bookArray')) || [];
+
+function updateLocalStorage() {
+  localStorage.setItem('bookArray', JSON.stringify(bookArray));
+}
+
+function displayBooks() {
+  const bookList = $.getElementById('booklist');
+
+  bookList.innerHTML = '';
+  bookArray.forEach((book, index) => {
+    const div = $.createElement('div');
+    div.setAttribute('id', 'div');
+    bookList.appendChild(div);
+    const titleB = $.createElement('h3');
+    titleB.innerHTML = `${book.book}`;
+    div.appendChild(titleB);
+    const authorB = $.createElement('h3');
+    authorB.innerHTML = `${book.author}`;
+    div.appendChild(authorB);
+    const buttonB = $.createElement('h3');
+    const removeButton = $.createElement('button');
+    removeButton.innerHTML = 'Remove';
+    removeButton.addEventListener('click', () => {
+      buttonB.parentElement.remove();
+      bookArray.splice(index, 1);
```

Comment:
- [Optional] Kindly it would be better if you use array methods that don't mutate the array, e.g. [filter](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/filter), [slice](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/slice)👍